prefect_census/utils.py

```diff
@@ -0,0 +1,20 @@
+"""Utilities for common interactions with the Census API"""
+from typing import Optional
+
+from httpx import HTTPStatusError
+
+
+def extract_user_message(error: HTTPStatusError) -> Optional[str]:
+    """
+    Extract user message from an error response from the Census API.
+
+    Args:
+        error: An HTTPStatusError raised by httpx
+
+    Returns:
+        The status from Census API response or None if a status cannot
+            be extracted.
+    """
+    response_payload = error.response.json()
+    user_message = response_payload.get("status", None)
+    return user_message
```

Comment:
If the body isn't JSON serializable, we can lose the error message all together. I recommend handling JSON decode exceptions so that the underlying error still bubbles up.
```suggestion
    try:
        response_payload = error.response.json()
        user_message = response_payload.get("message", None)
        return user_message
    except JSONDecodeError:
        pass
```